content/posts/2022/kerberos-all-steps-workflow.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Easily enable Kerberos for all the steps of a workflow"
+date: 2022-11-30T14:30:00+01:00
+tags:
+  - kerberos
+---
+
+Up until now, it was only possible to enable Kerberos support for a workflow on
+a step-by-step basis. This changes with REANA 0.9.0, which makes it easy to
+enable Kerberos authentication for an entire workflow by using only a single
+hint in its specification file.
+
+<!--more-->
+
+As an example, let's consider a workflow that uses Kerberos to access a folder
+on an external restricted storage backend like EOS to publish results there.
+This is how the `reana.yaml` specification file might look like:
+
+```yaml {hl_lines=[10]}
+workflow:
+  type: serial
+  specification:
+    steps:
+      - name: myfirststep
+        #...
```

Comment:
Perhaps use `- ...` so that the dashes will be there, for better analogy with the final step.

Reply:
I have replaced `#...` with:
```
environment: ...
commands:
  - ...
```